index.js

```diff
@@ -10,8 +10,8 @@ const consoleWidth = () => {
 
 program
   .version('0.2.0')
-  .arguments('<phrase>')
-  .action((slng) => {
+  .arguments('<phrase> [number]')
+  .action((slng, numberOfResults) => {
```

Comment:
We may be able to avoid the 2nd hunk of changes below if we assign a default value here. 

Would
```
.action(slng, numberOfResults = 3) => {
```
work?

Reply:
@jwu910 I don't think it's done that way. I looked at the examples and it's more like `.option('-r, --results [r]', 'Specify the number of results [r] to display', '3')`
[Source - Defaults Example](https://github.com/tj/commander.js/blob/master/examples/defaults)

** NOTE ** this uses the `.option` prop and not the action prop (comment on that below)

Reply:
Oh interesting. gotcha. I did see a bunch of options. I did also see a `{ isDefault: true }` on one of the examples. The defaults example might be more clear: https://github.com/tj/commander.js/blob/master/examples/defaults

Reply:
That default true is not what you're thinking

Reply:
I see that now. lol

Reply:
I know you said you see it now, but for reference:
isDefault just states that when you call an app with sub-commands and don't state a subcommand, the default one is performed
https://www.npmjs.com/package/commander#git-style-sub-commands

Reply:
Yeah thats where I saw it. It's a command thats called though, not assigning a value by default